x-pack/legacy/plugins/apm/index.ts

```diff
@@ -48,7 +48,10 @@ export const apm: LegacyPluginInitializer = kibana => {
       },
       hacks: ['plugins/apm/hacks/toggle_app_link_in_nav'],
       savedObjectSchemas: {
-        'apm-telemetry': {
+        'apm-services-telemetry': {
```

Comment:
I know I asked for this but I gotta ask: will this cause migration issues?

Reply:
It doesn't trigger a migration, as a it's a different object type, but it would remove saved objects of the old type. This won't be an issue as the object is ephemeral, and only reference when triggered by the usage collector.